sources/packages/backend/apps/api/src/services/application/application.service.ts

```diff
@@ -1420,16 +1428,66 @@ export class ApplicationService extends RecordDataModelService<Application> {
       },
       relations: {
         pirDeniedReasonId: true,
-        currentAssessment: true,
+        currentAssessment: { offering: true, disbursementSchedules: true },
         applicationException: true,
         location: true,
       },
       where: {
         id: applicationId,
+        applicationStatus: Not(ApplicationStatus.overwritten),
+        student: {
+          id: studentId,
+        },
+      },
+    });
+  }
+
+  /**
+   * Get enrolment details of an application.
+   * While fetching the enrolment details, the application
+   * expected to be in status Assessment or Enrolment or Completed.
+   * @param applicationId application.
+   * @param studentId applicant student.
+   * @returns application details
+   */
+  async getApplicationEnrolmentDetails(
+    applicationId: number,
+    studentId?: number,
+  ): Promise<Application> {
+    return this.repo.findOne({
+      select: {
+        id: true,
+        currentAssessment: {
+          id: true,
+          disbursementSchedules: {
+            id: true,
+            coeStatus: true,
+            disbursementDate: true,
+            disbursementScheduleStatus: true,
+            coeDeniedReason: { id: true, reason: true },
+            coeDeniedOtherDesc: true,
+          },
+        },
+      },
+      relations: {
+        currentAssessment: { disbursementSchedules: { coeDeniedReason: true } },
```

Comment:
Do we have an idea if this creates an inner or a left join?

Reply:
Type orm find* methods use left join. 
![image](https://user-images.githubusercontent.com/54600590/215587822-c8ee88a4-fc74-45fe-a8f2-ec13faa65c81.png)

Reply:
Are you sure that it is not based on the entity model? I was expecting a sample from the generated query.